specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/advisors.json

```diff
@@ -16,6 +16,147 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/advisors": {
+      "get": {
+        "tags": [
+          "ServerAdvisors"
+        ],
+        "description": "Gets a list of server advisors.",
+        "operationId": "ServerAdvisors_ListByServer",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of server advisors.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Advisor"
```

Comment:
the response should be a "value" array. Please model it similar to other list APIs in SQL namespace.

Reply:
Updated response to return AdvisorListResult.